packages/backend/src/services/tumblr/index.ts

```diff
@@ -19,313 +19,334 @@ import { uploadFromUrl } from "@/services/drive/upload-from-url.js";
+	apiLogger.warn("user created");
+	return { user: user, account: account, blogInfo: blogInfo };
 }
 
-export async function fetchTumblrFeed( user: User ) {
-  apiLogger.warn('fetching tumblr user '+ user.username + ' with UUID ' + user.tumblrUUID);
-  if(user.tumblrUUID) {
-    const responses = {
-      user: user.username,
-      new: []
-    }
-    const blogInfo = await getTumblrProfile(user.tumblrUUID);
-    if(!blogInfo) {
-      apiLogger.warn('user not found');
-      return;
-    }
-    let posts = await getTumblrPosts(blogInfo.name, 0);
-    posts = posts.reverse()
-    const lastUserUpdate = user.updatedAt;
-    const transforms = []
-
-    for(const post of posts) {
-        const postDate = new Date(post.isoDate);
-        if (!lastUserUpdate || !user.feedUpdatedAt || postDate > lastUserUpdate) {
-          responses.new.push(post);
-          let title = '';
-          if(post.title) {
-            const titleSansEllipsis = post.title.split('…')[0];
-            const strippedContent = post.content ? post.content.replace(/(<([^>]+)>)/gi, '') : null;
-            if( !strippedContent || strippedContent.indexOf(titleSansEllipsis) < 0) {
-              title = '<div class="tumblrTitle">' + sanitize(post.title) + '</div>';
-            } else {
-              title = '';
-            }
-          }
-          let { reblogTrail, postContent } = await transformToReblogs(post.content)
-          transforms.push( await transformToReblogs(post.content))
-
-          const reblogTrailBlock = reblogTrail.length?
-            '<div class="reblogTrail">' +
-              reblogTrail.map(formatReblogItem).join('<hr/>') +
-            '</div>' :
-            '';
-
-          const note = await create(user, {
-            createdAt: new Date(),
-            text:
-              '<div class="tumblrPost">'+
-                reblogTrailBlock +
-                title+
-                postContent +
-              "</div>",
-            apHashtags: post.categories,
-            noMentions: true,
-            url: post.link,
-          });
-        }
-    };
-
-    await Users.update(user.id, { feedUpdatedAt: new Date() });
-
-    return transforms; // response;
-  }
-  return null;
-}
\ No newline at end of file
+export async function fetchTumblrFeed(user: User) {
+	apiLogger.warn(
+		"fetching tumblr user " + user.username + " with UUID " + user.tumblrUUID,
+	);
+	if (user.tumblrUUID) {
+		const responses = {
+			user: user.username,
+			new: [],
+		};
+		const blogInfo = await getTumblrProfile(user.tumblrUUID);
+		if (!blogInfo) {
+			apiLogger.warn("user not found");
+			return;
+		}
+		let posts = await getTumblrPosts(blogInfo.name, 0);
+		posts = posts.reverse();
+		const lastUserUpdate = user.updatedAt;
+		const transforms = [];
+
+		for (const post of posts) {
+			const postDate = new Date(post.isoDate);
+			if (!lastUserUpdate || !user.feedUpdatedAt || postDate > lastUserUpdate) {
+				responses.new.push(post);
+				let title = "";
+
+				/*
```

Comment:
If the regexes here are too scary, I'll refactor them out. (I spent too long in the regex mines, it rotted my brain, I can no longer understand how normal people feel about them.)